packages/react/src/__tests__/Pagination/Pagination.test.tsx

```diff
@@ -24,4 +25,20 @@ describe('Pagination', () => {
     })
     expect(results).toHaveNoViolations()
   })
+
+  it('should render links instead of anchor tags with the renderPageLink prop', () => {
+    const {container} = HTMLRender(
+      <Pagination
+        pageCount={10}
+        currentPage={1}
+        renderPageLink={({content, key, className}) => (
+          <ReactRouterLikeLink key={key} className={className} to={`#${content}`}>
+            {content}
```

Comment:
the `content` could be a page number, but it could also be `Next` or `Previous` I think? Maybe we need an additional `pageNum` (or the actual `page` / `page.num` from `model.tsx`?)

Reply:
It would be nice `if `content` was `children` ... then the API could be

```tsx
renderPageLink={({pageNum, ...props}) => <Link {...props} to={`#${pageNum}`} />}
```

Reply:
Good idea!